website/docs/cli/commands/vendor/vendor-pull.md

```diff
@@ -54,16 +96,32 @@ Run `atmos vendor pull --help` to see all the available options
 ## Examples
 
 ```shell
+atmos vendor pull
 atmos vendor pull -c infra/account-map
 atmos vendor pull -c infra/vpc-flow-logs-bucket
 atmos vendor pull -c echo-server -t helmfile
 atmos vendor pull -c infra/account-map --dry-run
 ```
 
+<br/>
+
+:::note
+
+When executing the `atmos vendor pull` command, Atmos performs the following steps to decide which vendoring manifest to use:
+
+- If `vendor.yaml` manifest is found (in the directory from which the command is executed), Atmos will parse the file and execute the command
+  against it. If the flag `--component` is not specified, Atmos will vendor all the artifacts defined in the `vendor.yaml` manifest.
+  If the flag `--component` is passed in, Atmos will vendor only that component
+
```

Comment:
Will it also search for the `vendor.yaml` in the `components.terraform.base_path` from the `atmos.yaml`?

Reply:
I think it should also check the `base_path` in `atmos.yaml`